docs/DeveloperGuide.adoc

```diff
@@ -721,20 +721,67 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |user |add multiple phone numbers for a contact |easily find an alternate number to contact someone
+
+|`* * *` | user | undo/redo my last action | recover from mistakes
 
 |`* * *` |user |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
```

Comment:
Does your program need a 'delete' user story? I do not see the 'delete' functionality within the user stories.

Take note of crucial user stories such as 'CRUD'.

Correct me if I had mistakenly missed your 'delete' user story!